dashboard_data/AggregateByDOW(SerialChart).md

```diff
@@ -19,8 +19,9 @@ var fs = FeatureSetByPortalItem(
 // Group county level data by date. 
 var fs_gp = GroupBy(fs, ['date'], [{name: 'cases_by_day', expression: 'newcountconfirmed', statistic: 'SUM'}]);
 
-// Create array for holding features
+// Create array for holding features, feat object for populating array
 var features = [];
+var feat;
```

Comment:
need to remove the var feat in the loop then.  The variable is already declare, just use it in the loop

Reply:
Ah, whoops, thought I'd got all of them. Should be fixed now.